examples/distributed_data_classification_examples/domain_api_example.py

```diff
@@ -68,7 +70,7 @@ def main(args):
     domain_classifier = DomainClassifier(
         model_file_name=model_file_name,
         labels=labels,
-        filter_by=["Games", "Sports"],
+        # filter_by=["Games", "Sports"],
```

Comment:
LMK if I can help debug this. Are you planning to fix it in this PR or later?

Reply:
Will do, trying to fix it in this PR.

Reply:
Done